spec/vcr/RegistrationsController/get_/registrations/_step_name/with_a_valid_step_name/returns_a_success_response.yml

```diff
@@ -0,0 +1,49 @@
ZSBhcHBsaWNhbnRzLiBcbklmIHlvdSBhcmUgdXNpbmcgdGhlIEdldCBTY2hvb2wgRXhwZXJpZW5jZSBzZXJ2aWNlLCB3ZSB3aWxsIHNoYXJlIHNvbWUgb2YgeW91ciBwZXJzb25hbCBkYXRhIHRvIHRoZSBzdGFmZiB3aG8gbWFuYWdlIHNjaG9vbCBleHBlcmllbmNlIGF0IHRoZSBzY2hvb2wgd2hlcmUgeW91J3ZlIHJlcXVlc3RlZCBzY2hvb2wgZXhwZXJpZW5jZS4gVGhpcyBhbGxvd3MgdGhlbSB0byBwcm9jZXNzIHlvdXIgYXBwbGljYXRpb24gYW5kIGdldCBiYWNrIHRvIHlvdSBhYm91dCB5b3VyIHNjaG9vbCBleHBlcmllbmNlLiBcbkhvdyBsb25nIHdlIHdpbGwga2VlcCB5b3VyIHBlcnNvbmFsIGRhdGFcbllvdXIgcGVyc29uYWwgZGF0YSwgc3VwcGxpZWQgZm9yIHRoZSBwdXJwb3NlcyBvZiB0aGlzIHNlcnZpY2UsIHdpbGwgYmUgc3RvcmVkIGZvciA1IHllYXJzLiBXZSB3aWxsIGRlbGV0ZSB0aGlzIGluZm9ybWF0aW9uIGlmIHlvdSByZXF1ZXN0IHVzIHRvIGRvIHNvLlxuWW91ciBkYXRhIHByb3RlY3Rpb24gcmlnaHRzXG5Zb3UgaGF2ZSB0aGUgcmlnaHQ6XG7igKJcdHRvIGFzayB1cyBmb3IgYWNjZXNzIHRvIGluZm9ybWF0aW9uIGFib3V0IHlvdSB0aGF0IHdlIGhvbGRcbuKAolx0dG8gaGF2ZSB5b3VyIHBlcnNvbmFsIGRhdGEgcmVjdGlmaWVkIGlmIGl0IGlzIGluYWNjdXJhdGUgb3IgaW5jb21wbGV0ZVxu4oCiXHR0byByZXF1ZXN0IHRoZSBkZWxldGlvbiBvciByZW1vdmFsIG9mIHBlcnNvbmFsIGRhdGEgd2hlcmUgdGhlcmUgaXMgbm8gY29tcGVsbGluZyByZWFzb24gZm9yIGl0cyBjb250aW51ZWQgcHJvY2Vzc2luZ1xu4oCiXHR0byByZXN0cmljdCBvdXIgcHJvY2Vzc2luZyBvZiB5b3VyIHBlcnNvbmFsIGRhdGEgKGkuZS4gcGVybWl0dGluZyBpdHMgc3RvcmFnZSBidXQgbm8gZnVydGhlciBwcm9jZXNzaW5nKVxu4oCiXHR0byBvYmplY3QgdG8gZGlyZWN0IG1hcmtldGluZyAoaW5jbHVkaW5nIHByb2ZpbGluZykgYW5kIHByb2Nlc3NpbmcgZm9yIHRoZSBwdXJwb3NlcyBvZiBzY2llbnRpZmljL2hpc3RvcmljYWwgcmVzZWFyY2ggYW5kIHN0YXRpc3RpY3MgLSBidXQgdGhpcyB3aWxsIG1lYW4geW91IG5vIGxvbmdlciBjYW4gcmVjZWl2ZSB0aGUgc2VydmljZVxuXG5QbGVhc2UgY29udGFjdCB1cyBhdCBnZXRpbnRvdGVhY2hpbmcuaGVscGRlc2tAZWR1Y2F0aW9uLmdvdi51ayBhbmQgZW50ZXIgRGF0YSBQcm90ZWN0aW9uIGFzIGEgcmVmZXJlbmNlIHJlZ2FyZGluZyBhbnkgb2YgdGhlIHJpZ2h0cyBsaXN0ZWQgYWJvdmUuIEZvciB0aGUgRGF0YSBQcm90ZWN0aW9uIE9mZmljZXIgKERQTykgcGxlYXNlIGNvbnRhY3QgdXMgdmlhIGdvdi51ayBhbmQgbWFyayBpdCBmb3IgdGhlIGF0dGVudGlvbiBvZiB0aGUg4oCYRFBP4oCZLlxuSWYgeW91IG5lZWQgdG8gY29udGFjdCB1cyByZWdhcmRpbmcgYW55IG9mIHRoZSBhYm92ZSByaWdodHMsIHBsZWFzZSBkbyBzbyB2aWEgdGhlIEdldCBpbnRvIFRlYWNoaW5nIHNpdGVcbkZ1cnRoZXIgaW5mb3JtYXRpb24gYWJvdXQgeW91ciBkYXRhIHByb3RlY3Rpb24gcmlnaHRzIGFwcGVhcnMgb24gdGhlIEluZm9ybWF0aW9uIENvbW1pc3Npb25lcidzIHdlYnNpdGVcblJpZ2h0IHRvIGxvZGdlIGEgY29tcGxhaW50XG5JZiB5b3UgYXJlIHVuaGFwcHkgd2l0aCBvdXIgdXNlIG9mIHlvdXIgcGVyc29uYWwgZGF0YSwgcGxlYXNlIGxldCB1cyBrbm93IGJ5IGNvbnRhY3RpbmcgZ2V0aW50b3RlYWNoaW5nLmhlbHBkZXNrQGVkdWNhdGlvbi5nb3YudWtcbkFsdGVybmF0aXZlbHksIHlvdSBoYXZlIHRoZSByaWdodCB0byByYWlzZSBhbnkgY29uY2VybnMgd2l0aCB0aGUgSW5mb3JtYXRpb24gQ29tbWlzc2lvbmVyJ3MgT2ZmaWNlIChJQ08pIHZpYSB0aGVpciB3ZWJzaXRlLlxuTGFzdCB1cGRhdGVkXG5XZSBtYXkgbmVlZCB0byB1cGRhdGUgdGhpcyBwcml2YWN5IG5vdGljZSBwZXJpb2RpY2FsbHksIHNvIHdlIHJlY29tbWVuZCB0aGF0IHlvdSByZXZpc2l0IHRoaXMgaW5mb3JtYXRpb24gZnJvbSB0aW1lIHRvIHRpbWUuIFRoaXMgdmVyc2lvbiB3YXMgbGFzdCB1cGRhdGVkIG9uIDE0IEphbnVhcnkgMjAyMC5cbldlIHVzZSBjb29raWVzIHRvIG1ha2Ugb3VyIHNlcnZpY2VzIGVhc3ksIHVzZWZ1bCBhbmQgcmVsaWFibGUuIEZpbmQgb3V0IG1vcmUgb24gaG93IHdlIHVzZSBjb29raWVzLiIsImNyZWF0ZWRBdCI6IjIwMjAtMDEtMTNUMDk6NDQ6NTAiLCJpZCI6IjBhMjAzOTU2LWU5MzUtZWExMS1hODEzLTAwMGQzYTQ0YThlOSJ9
```

Comment:
Is there a setting in VCR that is encoding this to binary? it makes the diff impossible, this should just show the response body as JSON